lib/cli/lib/blueprint.js

```diff
@@ -121,27 +121,30 @@ export default class Blueprint {
   generate({ params, flags }) {
     let data = this.locals(params, flags);
     let dest = process.cwd();
-
-    walk(this.templateFiles).forEach((relativepath) => {
-      let absolutepath = path.resolve(path.join(this.templateFiles, relativepath));
-      if (isDir(absolutepath)) {
-        return null;
-      }
-
-      let filenameTemplate = template(relativepath, { interpolate: /__([\S]+)__/g });
-      let destRelativepath = filenameTemplate(data);
-      let destAbsolutepath = path.join(dest, destRelativepath);
-
-      if (fs.existsSync(destAbsolutepath)) {
-        return ui.info(`   ${ chalk.green('already exists') } ${ destRelativepath }`);
-      }
-
-      let contents = fs.readFileSync(absolutepath, 'utf-8');
-      let contentsTemplate = template(contents);
-      mkdirp.sync(path.dirname(destAbsolutepath));
-      fs.writeFileSync(destAbsolutepath, contentsTemplate(data));
-      ui.info(`  ${ chalk.green('create') } ${ destRelativepath }`);
-    });
+    let exists = fs.existsSync(this.templateFiles);
+
+    if (exists) {
```

Comment:
This just let's blueprints not include a `files` directory - am I understanding that correctly? In case I want a blueprint just for it's postInstall hook? If so - nice 👍